pkg/pillar/types/assignableadapters.go

```diff
@@ -209,9 +209,9 @@ func (aa *AssignableAdapters) LookupIoBundleGroup(group string) []*IoBundle {
 }
 
 // LookupIoBundleNet checks for IoNet*
```

Comment:
Please fix the comment. Yetus fails (note you can run make yetus in your workspace to check this)

Reply:
Done.